python/servo/testing_commands.py

```diff
@@ -615,6 +622,37 @@ def dromaeo_test_runner(self, tests, binary: str, bmf_output: str | None):
 
         return check_call([run_file, "|".join(tests), bin_path, base_dir, bmf_output])
 
+    def speedometer_to_bmf(self, speedometer: str, bmf_output: str | None):
+        output = dict()
+
+        def parse_speedometer_result(result):
+            if result["unit"] == "ms":
+                output[f"Speedometer/{result['name']}"] = {
+                    "latency": {  # speedometer has ms we need to convert to ns
```

Comment:
milliseconds to nano-seconds would be 10^6, right? is the comment wrong or the value.

Reply:
Lets make a separate follow-up PR to fix the wrong conversion. bencher.dev links to the github commit, so if we have it seperate it will be immedialty obvious why we have a 1000x "regression"